basis/formatting/formatting-docs.factor

```diff
@@ -78,8 +80,10 @@ HELP: printf
         "##12" }
     { $example
         "USING: formatting ;"
-        "1234 \"%+d\" printf"
-        "+1234" }
+        " 1234 \"%+d\\n\" printf"
```

Comment:
I wonder if it's not better to put a ``+`` instead of a space for positive integers?

Like Python:

```python
>>> '%+d' % 1234
'+1234'
>>> '%+d' % -1234
'-1234'
```